README.md

```diff
@@ -13,12 +13,14 @@ For complete documenation on Igluctl please refer to the [technical documentatio
 
 ## User Quickstart
 
-You can download igluctl archive from our Bintray, using following link:
+You can download pre-0.7.2 igluctl versions from our Bintray, using following link:
 
 ```
 http://dl.bintray.com/snowplow/snowplow-generic/igluctl_0.7.1.zip
 ```
 
+As of 0.7.2 release, artifacts are attached to GitHub releases.
```

Comment:
1. We need an example and a link to the page
2. Right now "as of 0.7.2..." is small note, but main pointer is to Bintray, whereas it should be vice versa - "You can download from github" and a small note that previous versions are available on Bintray

P.S. Don't forget to update docs website